.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: SpackBot CI
+
+on:
+  pull_request: []
+
+jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup black environment
+        run: |
+          conda create -c conda-forge -y -q --name black black
+
+      - name: Check formatting
+        run: |
+          export PATH="/usr/share/miniconda/bin:$PATH"
+          source activate black
+          black --check --diff spackbot
+
+      # Likely only maintainers will work on Spackbot
+      - name: Comment Pull Request
+        if: github.event_name == 'pull_request' && failure()
+        uses: marocchino/sticky-pull-request-comment@v1.1.0
```

Comment:
With this action, if I make my own fork of the repo, the action won't run on my PRs.

Reply:
Yes that's true, but not the entire action, just this step. I figured since only maintainers can really easily work on spack-bot it's usually going to be the case that we push to a branch here and this step will work ok.